db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devs, :array => true, :default => []
```

Comment:
@durandom They seem to be working nice together, have a look:
http://blog.plataformatec.com.br/2014/07/rails-4-and-postgresql-arrays/